safedelete/tests/test_soft_delete.py

```diff
@@ -145,6 +154,16 @@ def test_update_or_create_with_unique_field(self):
         self.assertEqual(obj.name, 'unique-test')
         self.assertEqual(created, False)
 
+    def test_update_or_create_with_unique_together_constraint(self):
+        # Create and soft-delete object
+        obj, created = UniqueTogetherSoftDeleteModel.objects.update_or_create(name='thor', team='avengers')
+        obj.delete()
+        # Update it and see if it fails
+        obj, created = UniqueTogetherSoftDeleteModel.objects.update_or_create(name='thor', team='avengers')
```

Comment:
@drnextgis what will happen when you delete the second object?

Reply:
It will be soft deleted.

Reply:
so only one record will be in the db, very nice and simple